### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by nspragg**
Refactor to:

- use let and const
- use lambda functions
- upgrade linter
- use ES6 classes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "sqs-consumer",
-  "version": "3.8.0",
```
Comment (cejast) [inline package.json]: should this be a major version?

Comment (nspragg) [reply package.json]: wasn't going to do release just yet but can do

**Comment by cejast** [comment]
LGTM 👍

**Comment by ealter** [comment]
Any updates on when you'll publish v4.0.0 on npm?

**Comment by nspragg** [comment]
@ealter - the most recent changes we made are maintenance improvements only - we're don't have a release planned at the moment.

**Pull request merged**